src/test/java/org/broadinstitute/hellbender/cmdline/argumentcollections/IntervalArgumentCollectionTest.java

```diff
@@ -110,6 +112,62 @@ public void testIntervalMergingRuleAdjacentNoMerge(IntervalArgumentCollection ia
+        iac.addToIntervalStrings("1:1-100");
+        iac.addToIntervalStrings("1:51-200");
+        iac.intervalMergingRule = IntervalMergingRule.NONE;
+        Assert.assertEquals(iac.getIntervals(hg19GenomeLocParser.getSequenceDictionary()).size(), 2);
+        Assert.assertEquals(iac.getIntervals(hg19GenomeLocParser.getSequenceDictionary()).get(0), new SimpleInterval("1", 1, 100));
+        Assert.assertEquals(iac.getIntervals(hg19GenomeLocParser.getSequenceDictionary()).get(1), new SimpleInterval("1", 51, 200));
+    }
+
+    @Test(dataProvider = "optionalOrNot")
+    public void testIntervalMergingRuleOverlappingNoMergeWithPadding(IntervalArgumentCollection iac){
+        iac.addToIntervalStrings("1:1-100");
+        iac.addToIntervalStrings("1:51-200");
+        iac.intervalMergingRule = IntervalMergingRule.NONE;
+        iac.intervalPadding = 10;
+        Assert.assertEquals(iac.getIntervals(hg19GenomeLocParser.getSequenceDictionary()).size(), 2);
+        Assert.assertEquals(iac.getIntervals(hg19GenomeLocParser.getSequenceDictionary()).get(0), new SimpleInterval("1", 1, 110));
+        Assert.assertEquals(iac.getIntervals(hg19GenomeLocParser.getSequenceDictionary()).get(1), new SimpleInterval("1", 41, 210));
+    }
+
+
+    @DataProvider(name = "NoMergeIllegalArguments")
+    public Object[][] NoMergeIllegalArguments(){
+        List<Object[]> argumentCollections = new ArrayList<>();
+
+        IntervalArgumentCollection iacXL = new RequiredIntervalArgumentCollection();
+        iacXL.excludeIntervalStrings.add("1:1-100");
+        argumentCollections.add(new Object[]{iacXL});
+
+        IntervalArgumentCollection iacMergingRule = new RequiredIntervalArgumentCollection();
+        iacMergingRule.intervalSetRule = IntervalSetRule.INTERSECTION;
+        argumentCollections.add(new Object[]{iacMergingRule});
+
+        return argumentCollections.toArray(new Object[][]{});
+    }
+
+    @Test(dataProvider = "NoMergeIllegalArguments", expectedExceptions = UserException.class)
+    public void testIntervalMergingRuleNoMergeIllegalArgumentCombinations(IntervalArgumentCollection iac) {
+        iac.addToIntervalStrings("1:1-100");
+        iac.addToIntervalStrings("1:51-200");
+        iac.intervalMergingRule = IntervalMergingRule.NONE;
+        iac.getIntervals(hg19GenomeLocParser.getSequenceDictionary());
+    }
+
+    @Test(dataProvider = "optionalOrNot")
+    public void getUserSpecifiedIntervalsWithoutMerging(IntervalArgumentCollection iac){
+        iac.addToIntervalStrings("1:1-100");
+        iac.addToIntervalStrings("1:90-200");
+        iac.intervalSetRule = IntervalSetRule.INTERSECTION;
+        Assert.assertEquals(iac.getIntervals(hg19GenomeLocParser.getSequenceDictionary()), Arrays.asList(new SimpleInterval("1", 90, 100)));
+        Assert.assertEquals(iac.getSpecifiedIntervalsWithoutMerging(hg19GenomeLocParser.getSequenceDictionary()), Arrays.asList(new SimpleInterval("1", 1, 100), new SimpleInterval("1", 90, 200)));
```

Comment:
A few more direct tests on the `getSpecifiedIntervalsWithoutMerging()` method would be a good idea.

Reply:
These tests have been adapted to test that method directly now.